src/Microsoft.ML.Onnx/TransformerChainOnnxConverter.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Data;
+using Microsoft.ML.UniversalModelFormat.Onnx;
+
+namespace Microsoft.ML.Model.Onnx
+{
+    public class TransformerChainOnnxConverter
+    {
+        public static ModelProto Convert<T>(TransformerChain<T> chain, IDataView inputData, HashSet<string> inputColumnNamesToDrop=null, HashSet<string> outputColumnNamesToDrop=null) where T : class, ITransformer
```

Comment:
>ModelProto [](start = 22, length = 10)

Our typical approach for new capabilities like this is that we'd, as much as possible, make operations like this. How do we current expose saving an `ITransformer`? Through `ModelOperationsCatalog`. This should have likewise been an extension method on `ModelOperationsCatalog`. (And, like all other things like this, in namespace `Microsoft.ML`.) That was this functionality gets discoverable through `MLContext`, which is our strategy for things like this so far.

Reply:
A fix will be pushed soon.

---
In reply to: [245215676](https://github.com/dotnet/machinelearning/pull/2013#discussion_r245215676) [](ancestors = 245215676)